resources/hiveclients/python/hiveserver1-client.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of fiware-cygnus (FI-WARE project).
+#
+# fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+# General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+# fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+
+# imports
+import sys
+import hive_utils
+from hive_service.ttypes import HiveServerException
+
+# get the input parameters
+if len(sys.argv) != 6:
+    print 'Usage: python hiveserve1-client.py <hive_host> <hive_port> <db_name> <hadoop_user> <hadoop_password>'
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/cfc371eee3d79575e3021bb4c7772636112380ae